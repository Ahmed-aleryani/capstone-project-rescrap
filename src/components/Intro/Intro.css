.intro {
  background-image: url(../../images/Component\ 2.jpg);
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;
}
.intro-content {
  width: 32rem;
}
button {
  background-color: white;
  color: green;
  border-radius: 9999px;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 1rem;
  padding-right: 1rem;
}

@media only screen and (max-width: 920px) {
  .intro-content {
    width: 24rem;
  }
}
@media only screen and (max-width: 680px) {
  .intro-content {
    width: 16rem;
  }
}
