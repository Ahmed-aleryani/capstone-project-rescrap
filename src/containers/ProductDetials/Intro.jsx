import React from 'react';
import SubTitle from '../../components/SubTitle/SubTitle';
import Title from '../../components/Title/Title';
const Intro = (props) => {
  return (
    <div>
      <div className="flex flex-wrap justify-center ">
        <div className="w-full md:w-1/2 lg:w-1/2 flex justify-center shadow-md mb-2">
        <div className="flex justify-center w-full mx-6 my-4">
          <img
            src={props.imgSrc}
            alt={props.description}
            className="justify-self-center"
          />
        </div>
        </div>
        <div className="flex items-center justify-center w-full md:w-1/2 lg:w-1/2">
        <div className="flex flex-col justify-center overflow-hidden  w-1/2 p-2 ">
          <Title
            classes="uppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
            text={props.productName}
          />

          <div className="text-base flex   my-3   text-gray-700">
            <p>
          {props.description}
            </p>
          </div>
          <SubTitle
            text={props.price}
            classes=" text-green-200 uppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
          />
        </div>
        </div>
      </div>
    </div>
  );
};
export default Intro;
