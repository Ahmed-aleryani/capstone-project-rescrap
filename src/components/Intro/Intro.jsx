import React from 'react';
import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
import Navbar from '../Navbar/Navbar';
import Title from '../Title/Title';
import Description from '../Description/Description';
import Butten from '../Button/Button';
import './Intro.css'

const Intro = () => {
    return ( <div className="intro h-screen">
        <Router>
      <Navbar />
      <Switch>
        <Route path="/about-us">About Us</Route>
        <Route path="/shop">Shop</Route>
        <Route path="/add-product">Add Product</Route>
        <Route path="/contact-us">Contact Us</Route>
        <Route path="/"></Route>
      </Switch>
    </Router>
    <div className="intro-content p-12 py-24">
        <Title classes=" text-green-500 my-4" text="Plastic Recycling Marketplace"/>
        <Description classes=" text-lg my-4" text="Connect and trade directly with suppliers & buyers of recyclable plastics" />
        <Butten btnClasses=" bg-green-500 text-white my-4" btnClickHandler='' btnText="Sign up" />
    </div> </div>);
}
 
export default Intro;