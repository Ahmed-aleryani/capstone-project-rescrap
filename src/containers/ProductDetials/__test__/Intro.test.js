import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../containers/PorudctDetails/Intro';

it('matches snapshot', () => {
  const y = renderer.create(<Intro />).toJSON();
  expect(y).toMatchSnapshot();
