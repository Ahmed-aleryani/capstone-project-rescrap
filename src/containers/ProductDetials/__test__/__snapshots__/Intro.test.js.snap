// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center "
  >
    <div
      className="w-full md:w-1/2 lg:w-1/2 flex justify-center shadow-md mb-2"
    >
      <div
        className="flex justify-center w-full mx-6 my-4"
      >
        <img
          className="justify-self-center"
        />
      </div>
    </div>
    <div
      className="flex items-center justify-center w-full md:w-1/2 lg:w-1/2"
    >
      <div
        className="flex flex-col justify-center overflow-hidden  w-1/2 p-2 "
      >
        <h1
          className="text-3xl normaluppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
        />
        <div
          className="text-base flex   my-3   text-gray-700"
        >
          <p />
        </div>
        <h2
          className="text-2xl thin text-green-200 uppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
        />
      </div>
    </div>
  </div>
</div>
`;
